app/assets/stylesheets/basic.scss

```diff
@@ -1,19 +1,28 @@
 
 @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+@import url(https://fonts.googleapis.com/css?family=Muli:400,300italic,300);
+@import url(https://fonts.googleapis.com/css?family=Alegreya+Sans+SC:800);
 
  /************** BASIC STYLING *****************************/
 
-#content {
-  background-color: #FFF;
-  margin-bottom: 6rem;
-  margin-top: 6rem;
-}
+$color: #EC4339;
+
+ .title-box {
+   background-color: $color;
+   color: #FFFFFF;
+   padding: 10px;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, font-size, font-weight, letter-spacing, padding